charts/timed/values.yaml

```
@@ -229,20 +241,15 @@ ingress:
   #      - timed-test.example.com
 
 postgresql:
-  # -- Enable PostgreSQL for persistence
+  # -- Enable PostgreSQL Helm Chart (https://bitnami.com/stack/postgresql) for persistence
   enabled: true
-  # -- Use an existing PostgreSQL instance
-  existingHost: ""
-  # -- PostgreSQL database name
-  postgresqlDatabase: timed
-  # -- PostgreSQL user name
-  postgresqlUsername: postgres
-  # -- Set a password for PostgreSQL
-  # postgresqlPassword: s3cr3t
-  image:
-    # Use Postgresql 12
-    # -- PostgreSQL image version to use
-    tag: 12.3.0
+  auth:
+    # -- PostgreSQL database name
+    database: timed
+    # -- PostgreSQL user name
+    username: postgres
```

most likely the initially created user needs renaming as well if we use password instead of postgres-password:
```suggestion
    username: timed
```

---

took some time to understand the testing-setup. it may be that the "migrate.py" task might fail because of downgrade of privileges. It could also be that the "migrate.py" already wants to create a user named "timed" (that would be an easy fix).
lets see. hopeful it will just work :D

---

woohoo :) it werked :tada: